package-examples/helloworld-tshirt/Kptfile

```diff
@@ -0,0 +1,10 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: helloworld-tshirt
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/apply-setters:unstable
+      configMap:
+        tshirt-size: 'small'
```

Comment:
Some examples have quotes and some don't. Better to maintain consistency.

Reply:
fixed